### DISCUSSION TITLE
Wrong frequency ?

### DISCUSSION THREAD

**Discussion by joni-swe** [jekhokie/raspberry-noaa-v2#527] [Q&A]
Hello!
I´m trying to recieve images from noaa satelites, with bad results.
My Rpi is runniing RaspberryNoaa_V2 and the log file shows this:

NOAA 15
09-04-2022 08:16 /home/pi/raspberry-noaa-v2/scripts/audio_processors/noaa_record_rtl_fm.sh INFO : Recording at 137.6200 MHz...
Found 1 device(s):
0: Realtek, RTL2838UHIDIR, SN: 00000001
Using device 0: Generic RTL2832U OEM
Found Rafael Micro R820T tuner
Tuner gain set to 15.70 dB.
Tuned to 138100000 Hz.
Oversampling input by: 32x.
Oversampling output by: 1x.
Buffer size: 4.27ms
Allocating 15 zero-copy buffers
Sampling at 1920000 S/s.
Output at 60000 Hz.
Signal caught, exiting!

Then it´s recording but the frequency 138.1MHz is way off, why?
The doppler effect should be only a few kHz not 480kHz?
Is this correct?

**Comment by dom-robinson** [comment]
looks wierd - where is this log from? Output.log doesnt usually include the Tuned to (as far as i can see from my running unit).

Also a quick thing to try is flip between rtl_fm and gnuradio mode.

**Comment by joni-swe** [comment]
I use this command to follow what happens.
tail -f /var/log/raspberry-noaa-v2/output.log
i will try to use gnuradio instead, but can I use gnuradio with RTL stick?

**Reply by dandspach** [reply]
Did you ever get this resolved?  On my images, I show the recorded freq different than the sat freq, yet images are still ok.

In the image below, the recorded freq should be around 137.1000 MHz, yet the recorded freq looks like 2150 HZ.  I opened an 'issue' on it, but have not seen anyone respond or comment yet.

![image](https://github.com/jekhokie/raspberry-noaa-v2/assets/12740373/e65f1cc2-8e2a-4860-a21b-8a199bcddbcd)
